main/src/cgeo/geocaching/utils/AndroidRx2Utils.java

```diff
@@ -0,0 +1,99 @@
+package cgeo.geocaching.utils;
+
+import android.app.Activity;
+import android.os.HandlerThread;
+import android.os.Looper;
+import android.os.Process;
+
+import java.lang.ref.WeakReference;
+import java.util.concurrent.Callable;
+import java.util.concurrent.Executors;
+
+import io.reactivex.Maybe;
+import io.reactivex.Observable;
+import io.reactivex.Scheduler;
+import io.reactivex.Single;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.functions.Consumer;
+import io.reactivex.functions.Predicate;
+import io.reactivex.internal.schedulers.RxThreadFactory;
+import io.reactivex.schedulers.Schedulers;
+
+public class AndroidRx2Utils {
+
+    public static final Scheduler computationScheduler = Schedulers.computation();
+
+    public static final Scheduler networkScheduler = Schedulers.from(Executors.newFixedThreadPool(10, new RxThreadFactory("network-")));
```

Comment:
Oh. So you create all the memory leaks where a thread called "network-x" is mentioned? ;) See #5850. I always thought those threads are from the framework. With this new knowledge gained, is there anything that we can and should do to avoid the leaks referring to these threads?